benchmarks/common.py

```diff
@@ -1059,8 +1148,8 @@ def main(runner, original_dir=None):
         runner.skip_models.clear()
 
     experiment = null_experiment
+    global current_name, current_device, output_filename, optimize_ctx
```

Comment:
Do we need optimize_ctx to be global?

Reply:
well, i needed it passed into my experiment. i would be ok threading it through instead of making it global, but i wasn't sure which way to go.

Reply:
Its ok. We can deal with this in a separate PR.